apps/console-lite/src/app/lib/vega-connectors.ts

```diff
@@ -1,7 +1,9 @@
-import { RestConnector } from '@vegaprotocol/wallet';
+import { RestConnector, JsonRpcConnector } from '@vegaprotocol/wallet';
 
 export const rest = new RestConnector();
+export const jsonRpc = new JsonRpcConnector();
 
 export const Connectors = {
   rest,
+  jsonRpc,
```

Comment:
Question: is this required to run the new wallet?

Reply:
Yes, it was missing in console-lite connector.